Orange/widgets/visualize/owmosaic.py

```diff
@@ -337,7 +338,7 @@ def setData(self, data):
 
         if any(attr.is_continuous for attr in self.data.domain):
             self.information(0, "Data contains continuous variables. "
-                                "Discretize the data to use them.")
+                                "Default Discretization by Equal Frequency.")
```

Comment:
Let us remove this message: the user knows that these features are numeric and (s)he can see that intervals are used.